tests/scripts/pgocheck/pefile/ordlookup/__init__.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+from . import ws2_32
+from . import oleaut32
+
+'''
+A small module for keeping a database of ordinal to symbol
+mappings for DLLs which frequently get linked without symbolic
+infoz.
+'''
+
+ords = {
+    b'ws2_32.dll': ws2_32.ord_names,
```

Comment:
At test time I wasn't sure if this tool would be available, and if it is I am not sure if the output will be stable enough across versions to parse. At least that's why I decided to go with python in the first place, because I knew python 2.7 would be. Do you think these concerns are unfounded, or that the "cost" of having pefile here outweighs them?